src/gui/Application.cpp

```diff
@@ -115,31 +115,42 @@ Application::Application(int& argc, char** argv)
     lock->setStaleLockTime(0);
     lock->tryLock();
     switch (lock->error()) {
-    case QLockFile::NoError:
-        server.setSocketOptions(QLocalServer::UserAccessOption);
-        server.listen(socketName);
-        connect(&server, SIGNAL(newConnection()), this, SIGNAL(anotherInstanceStarted()));
-        break;
-    case QLockFile::LockFailedError: {
-        alreadyRunning = true;
-        // notify the other instance
-        // try several times, in case the other instance is still starting up
-        if (config()->get("SingleInstance").toBool()) {
-            QLocalSocket client;
-            for (int i = 0; i < 3; i++) {
-                client.connectToServer(socketName);
-                if (client.waitForConnected(150)) {
-                    client.abort();
-                    break;
+        case QLockFile::NoError:
```

Comment:
@droidmonkey I actually think not indenting the case labels is the preferred style in the project. At least that's the conclusion I came to when I was writing the `clang-format` file for the project. Either way, it'll make it easier to review if you preserve the previous style, at least in this PR.